File: spring/userMS/src/main/java/tn/inetum/userDoc/entity/Signataire.java
package tn.inetum.userDoc.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;




@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Signataire {


    @Id
    private String idSig;
    private String lnameSig;
    private String fnameSig;
    private String emailSig;
    private String passwordSig;
    private String telSig;


}

File: angular/src/app/document/models/signataire.ts
import { Element } from "./element";

export class Signataire {
  constructor(
    public idSig: string,
    public lnameSig: string,
    public fnameSig: string,
    public emailSig: string,
    public passwordSig: string,
    public telSig: string,
    public color: string,
    public elements: any[]
  ) {}
}
